doc/releases/changelog-dev.md

```diff
@@ -275,6 +275,11 @@
 
 <h3>Breaking changes 💔</h3>
 
+* The functions `qml.transforms.one_qubit_decomposition`, `qml.transforms.two_qubit_decomposition`, 
+  `qml.transforms.sk_decomposition` were moved to respectively, `qml.ops.one_qubit_decomposition`, `qml.ops.two_qubit_decomposition`, 
+  `qml.ops.sk_decomposition`.
+  [(#4906)](https://github.com/PennyLaneAI/pennylane/pull/4906)
```

Comment:
Can we have a one-cycle deprecation for this? I worry that these are slightly more popular functions and moving them in a breaking way could be more noticed. So specifically we could do:
- We do the move as it is in this PR
- Plus, `qml.transforms.one_qubit_decomposition` and `qml.transforms.two_qubit_decomposition` are simple wrappers that use the new location as well as providing a deprecation warning. (`sk_decomposition` is unreleased so no deprecation)

Reply:
Added :+1:

Reply:
@rmoyard do you think we still need to list this as a breaking change? Since `qml.transforms.one_qubit_decomposition` is still available, it's just that the underlying code has been moved.

Reply:
You are correct, we can readd this entry next release when the `qml.transforms` path is removed.

Reply:
Great, I'll do that in a separate PR.